mount/lookup_linux_test.go

```diff
@@ -28,7 +29,6 @@ import (
 	"github.com/containerd/continuity/testutil"
 	"github.com/containerd/continuity/testutil/loopback"
 	"github.com/stretchr/testify/assert"
-	exec "golang.org/x/sys/execabs"
```

Comment:
unnecessary change

Reply:
I was on the fence on this (this was spurred by some of my feedback). It seems they added the changes from execabs to os/exec in 1.19, and @thaJeztah had a PR open to revert them. The discussion stalled with waiting for 2.0 to go through with it.

Reply:
Which we're in the middle of working towards 🤓

Reply:
If reducing size of this PR increases its chances to get in, I don't care. `execabs` -> `exec` pass can happen independently on the whole codebase. We're down to -18loc!

Reply:
I'm with ya, keeping the diff scoped to just this is fine with me